site/persistent-volumes.md

```diff
@@ -0,0 +1,166 @@
+
+
+# Persistent Volumes
+
+This guide outlines how to set up a Kubernetes persistent volume (PV) and persistent volume claim (PVC), which can then be used in a domain custom resource as a persistent storage for the WebLogic domain home or log files. A PV and PVC can be shared by multiple WebLogic domains or dedicated to a particular domain.
+
+
+## Prerequisites
+
+The following prerequisites must be fulfilled before proceeding with the creation of the persistent volume.
+* Create a Kubernetes namespace for the persistent volume claim unless the intention is to use the default namespace. Note that a PVC has to be in the same namespace as the domain resource that uses it.
+* Make sure that the host directory that will be used as the persistent volume already exists and has the appropriate file permissions set.
+
+
+# YAML templates
+
+Persistent volumes and claims and their properties are described in YAML files. For each PV, you should create one PV YAML and one PVC YAML file. Below you can find two YAML tempaltes, one for the PV and one for the PVC. Each template references the following configuration parameters.
```

Comment:
Below you can find two YAML tempaltes (typo)->Below you can find two YAML templates